src/hashtables.jl

```diff
@@ -19,18 +19,30 @@
 type ParamsTaylorN
     maxOrder :: Int
     numVars  :: Int
+    variable_names :: Array{UTF8String,1}
```

Comment:
I think a reasonable default should be fixed, e.g., as in line 171 below. Something like
```julia
     variable_names :: UTF8String[string("x", subscriptify(i)) for i in 1:numVars]
```

Incidentally, I recall that you proposed to change maxOrder -> maxorder, and numVars -numvars. Am I right?